src/components/AdminPageTable/ExportLogsModal.jsx

```diff
@@ -11,8 +11,9 @@ import {
   ModalFooter,
   useToast,
   useDisclosure,
+  Icon,
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Icon' is defined but never used.